src/backend/expungeservice/models/helpers/record_summarizer.py

```diff
@@ -53,15 +54,27 @@ def summarize(record) -> RecordSummary:
         for county, balance in county_balances.items():
             county_balances_list.append(CountyBalance(county, round(balance, 2)))
         total_charges = len(record.charges)
-        eligible_charges = [
+        eligible_charges_now = [
             c.name
             for c in record.charges
             if c.expungement_result.charge_eligibility.status == ChargeEligibilityStatus.ELIGIBLE_NOW
         ]
+        eligible_charges_by_date = [["now",eligible_charges_now]]
```

Comment:
```suggestion
        eligible_charges_by_date = [("now",eligible_charges_now)]
```

`List`s of fixed lengths should be `Tuple`s